datamodels/2.x/itop-core-update/dictionaries/en.dict.itop-core-update.php

```diff
@@ -54,6 +54,7 @@
 
     'iTopUpdate:UI:Status' => 'Status',
     'iTopUpdate:UI:Action' => 'Update',
+    'iTopUpdate:UI:Setup' => 'iTop Setup',
```

Comment:
```suggestion
    'iTopUpdate:UI:Setup' => ITOP_APPLICATION_SHORT.' Setup',
```

Use the application name constant instead of 'iTop' please

Reply:
Fixed in 361ffe90